src/composables/Project.js

```diff
@@ -1,10 +1,72 @@
 import { randomHexString } from 'src/composables/Random';
 
+export const PROJECT_STORAGE_KEY = 'projects';
+
+/**
+ * @typedef {Object} Project
+ * @property {string} id project id
+ */
+
 /**
  * Create a project with generated id.
  *
- * @returns {{Object}} Project object with generated id.
+ * @returns {Project} Project object with generated id.
  */
-export function createProject() {
+export function createProjectTemplate() {
```

Comment:
why not register the project newly created in the local storage ? And then ...

Reply:
It's linked to how I generate the link for a, yet unexisting, project.

A "fake/template" project is made for the project creation button/form, when I click on the button/complete the form the, then created, project is saved.

Reply:
So, this is how I see it :
* createProjectTemplate() -> new Project
* createProject(Project) : add the project to the localstorage -> nothing (To Do)
* updateProject(Project) : update the project in the localstorage -> nothing (saveProject to rename)

Like this, it look like CRUD/API convention a bit more.

Reply:
`createProject(Project)` should be created

Reply:
after a talk with @Zorin95670 , we agree on :
keep `createProjectTemplate`
keep `saveProject ` (sorry for the useless change for `updateProject`)